packages/p2p/src/components/my-ads/quick-add-modal.scss

```
@@ -28,7 +28,13 @@
         display: flex;
     }
 
-    &--message {
-        margin-top: 1rem;
+    &--info {
+        margin-bottom: 1.3rem;
+    }
+
+    &--pointer-events {
+        & .dc-modal-header__close .dc-icon {
```

@adrienne-deriv please check if it's gonna work the same way without &:
```suggestion
        .dc-modal-header__close .dc-icon {
```
it should🙂